### DISCUSSION TITLE
Routing multiple web audio sources

### DISCUSSION THREAD

**Discussion by isul23k** [ExistentialAudio/BlackHole#739] [Q&A]
Hi. I'm looking to find a way to route multiple browsers to independent channel inputs. Someone gave me the clue that BlackHole might be able to do this?

**Comment by devinroth** [comment]
Sorry. Can’t be done with BlackHole. You’ll need to use Rogue Amoeba’s Loopback for that.

Devin

**Comment by isul23k** [comment]
From what I can see, Loopback has the possibilities of playing multiple applications but still it's limited to one channel out per app. Now I'm wondering if there is a way of rewiring individual browsers to act as an independent application, whether I need to think about problem-solving at the source or at the interface, I'm totally lost.
